deployment/cdk/opensearch-service-migration/lib/opensearch-domain-stack.ts

```diff
@@ -121,7 +121,9 @@ export class OpenSearchDomainStack extends Stack {
     if (adminUserName) {
         targetCluster.basic_auth = new ClusterBasicAuth({ username: adminUserName, password_from_secret_arn: adminUserSecret?.secretArn })
     } else {
-      targetCluster.no_auth = ''
+        targetCluster.sigv4 = new ClusterSigV4Auth();
+        targetCluster.sigv4.region = this.region;
+        targetCluster.sigv4.service = "es";
```

Comment:
Can this not be overridden?

Reply:
Well, if we're using this stack, we're creating the domain so we know it's `es`. Do you have any cases in mind where it might be something else?